test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -82,11 +84,27 @@ def test_tensorflow_telemetry_gpu(tensorflow_training, ec2_connection, gpu_only)
 def test_tensorflow_telemetry_cpu(tensorflow_training, ec2_connection, cpu_only):
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_TELEMETRY_CMD)
 
+
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_keras_horovod_amp(tensorflow_training, ec2_connection, gpu_only):
+    if is_tf1(tensorflow_training) or is_tf20(tensorflow_training):
+        pytest.skip("This test is for TF2.1 and later only") # https://github.com/tensorflow/tensorflow/issues/33484#issuecomment-555299647
+    execute_ec2_training_test(ec2_connection, tensorflow_training, TF_KERAS_HVD_CMD_AMP)
+
+
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_keras_horovod_fp32(tensorflow_training, ec2_connection, gpu_only):
```

Comment:
Just to confirm that fp32 test is compatible with TF2.0 and newer image like TF2.2 as well ?

Reply:
Yes. TF2 images all should pass FP32 test